### PR TITLE
BaSP-TG-84: Task redux - Create, update and delete

### PR DESCRIPTION
**Pull request by MaxiG-dev**
- Create constants to create, update and delete a task
- Create actions to create, update and delete a task
- Create thunks to create, update and delete a task
- Complete task reducer to create, update and delete a task
- Update modal message and modal confirm functions

### CODE REVIEW

File: src/Components/Shared/Modal/ModalConfirm/index.js [hunk 1]
```diff
@@ -35,3 +28,33 @@ const ModalConfirm = ({ show, closeModal, modalFunction, modalTitle, modalConten
...
+//               <Buttons variant="primary" name="Confirm" onClick={() => onConfirm()} />
+//             </span>
+//           </div>
+//         </div>
+//       </div>
+//     )
+//   );
+// };
+
+// export default ModalConfirm;
```
Comment (CristianBucci) [inline src/Components/Shared/Modal/ModalConfirm/index.js:60]: Remove this comented lines!

File: src/Components/Shared/Modal/ModalMessage/index.js [hunk 2]
```diff
@@ -33,3 +23,39 @@ const ModalMessage = ({ show, modalFunction, modalTitle, modalContent }) => {
...
+//           <div className={styles.content}>
+//             <span>{modalContent}</span>
+//           </div>
+//         </div>
+//       </div>
+//     )
+//   );
+// };
+
+// export default ModalMessage;
```
Comment (CristianBucci) [inline src/Components/Shared/Modal/ModalMessage/index.js:61]: Remove this commented lines!

File: src/Components/Shared/Modal/ModalConfirm/index.js (hunk 1 above)
Comment (MaxiG-dev) [reply src/Components/Shared/Modal/ModalConfirm/index.js]: Done!

**Review by melissabonillasellanes — APPROVED** [review]
Everything looks fine!

**Review by CristianBucci — APPROVED** [review]
Everything looks correct!

**Review by ignacioaristo — APPROVED** [review]

**Review by matescarabino — APPROVED** [review]
LGTM

**Pull request merged**